docker/install-dynamic-plugins.py

```diff
@@ -95,8 +137,30 @@ def main():
                 file.extract(member, path=directory)
             elif member.isdir():
                 print('\t\tSkipping directory entry', member.name, flush=True)
+            elif member.islnk() or member.issym():
+                if not member.linkpath.startswith('package/'):
+                  raise InstallException('NPM package archive contains a link outside of the archive: ' + member.name + ' -> ' + member.linkpath)
+
+                member.name = member.name[8:]
+                member.linkpath = member.linkpath[8:]
```

Comment:
Instead of using string slicing lets use a regex instead. This way it's easier to track if we change the `package/` location. We should also update line 136 as well. 

```suggestion
import re

locationRegex = r"package\/(.*)"

...

                member.name = re.search(r"package\/(.*)", member.name).group(1)
                member.linkpath = re.search(r"package\/(.*)", member.linkpath).group(1)
```

Reply:
There's not reason to change the `package/` location. It's part of how archives built by `npm pack` are structured.
That's why refuse any archive member which doesn't start with `package`.

Reply:
Can we use a regex instead of array slicing to assign `name` and `linkpath`? I think it's easier to read on a passthrough.

Reply:
Fixed in commit 47e9246829cdbaa30ae780ab7fb1fe7d8d75cfd1 as discussed with the `removeprefix` function instead of a regexp (even more readable)